app/controllers/creatives_controller.rb

```diff
@@ -29,13 +29,15 @@ def new
 
   def create
     @creative = current_user.creatives.build(creative_params)
+    images_valid = creative_image_params_validation.valid?
 
     respond_to do |format|
-      if @creative.save
+      if @creative.save && images_valid
```

Comment:
Need to do it like this bc if we call `creative_image_params_validation.valid?` on this line, and there are errors that prevent the creative from saving, it won't call `.valid?` which means the errors will not get added to the object which would make `creative_image_params_validation.errors` empty. Similarly, if we were to do `creative_image_params_validation.valid? && @creative.save`, any errors on the creative itself would not be added to the error messages.